app/models/patient.rb

```diff
@@ -22,6 +25,27 @@ class Patient
     scope line.downcase.to_sym, -> { where(:_line.in => [line]) }
   end
 
+  CSV_EXPORT_FIELDS = { "BSON ID" => :id,
```

Comment:
Let's chuck this in a concern, in `app/models/concerns/exportable.rb`, to match our new-ish patterns elsewhere.